src/lib/jsonParseV2/parseContextAttribute.cpp

```diff
@@ -153,8 +153,8 @@ static std::string parseContextAttributeObject
     }
     else  // ERROR
     {
-      LM_W(("Bad Input (unrecognized property for ContextAttribute - '%s')", name.c_str()));
-      return "unrecognized property for context attribute";
+      alarmMgr.badInput(clientIp, "JSON parse error");
+      return "JSON parse error in context attribute";
```

Comment:
This is not a "Parse Error" !
The current error message is quite good, and should not be changed.
Add a call to the alarm manager?  Sure, why not?
But, let's not lie about what went wrong!
This is NOT a parse error, but an unexpected (and not welcome) item

Reply:
Yes, you are right! I have reverted my changes in https://github.com/telefonicaid/fiware-orion/pull/4049/commits/5acdc12ec5d93282bd71423d4572167778759baa and added alarm for badinput.